.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 python:
         - "2.7"
         - "3.5"
+        - "3.6"
```

Comment:
We have no plans of supporting `3.6` currently.

Reply:
Got it, I was working with 3.6 so I added will resolve any issues.